src/smartpeak/include/SmartPeak/core/Filenames.h

```diff
@@ -54,6 +54,7 @@ namespace SmartPeak
     std::string quantitationMethods_csv_i;
     std::string standardsConcentrations_csv_i;
     std::string referenceData_csv_i;
+    std::string selectiveDilutions_csv_i;
```

Comment:
I would recommend renaming to `selectDilutions`

Reply:
indeed.